ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
-
 import {
   COLORS,
   FONT_WEIGHT,
   TYPOGRAPHY,
 } from '../../../../helpers/constants/design-system';
-import { roundToDecimalPlacesRemovingExtraZeroes } from '../../../../helpers/utils/util';
+import { isNullish } from '../../../../helpers/utils/util';
+import { formatGasFee } from '../../../../helpers/utils/gas';
 import { useGasFeeContext } from '../../../../contexts/gasFee';
 import I18nValue from '../../../ui/i18n-value';
 import Typography from '../../../ui/typography/typography';
-
-import { BaseFeeTooltip } from './tooltips';
-import LatestPriorityFeeField from './latest-priority-fee-field';
+import { BaseFeeTooltip, PriorityFeeTooltip } from './tooltips';
 import StatusSlider from './status-slider';
 
 const NetworkStatistics = () => {
   const { gasFeeEstimates } = useGasFeeContext();
+  const formattedLatestBaseFee = formatGasFee(
```

Comment:
This code got refactored slightly to take advantage of my new `formatGasFee` function, and to move LatestPriorityFeeField into this component.